cyences_app_for_splunk/default/data/ui/views/cs_overview.xml

```diff
@@ -619,6 +605,34 @@
       </table>
     </panel>
   </row>
+  <row>
+    <panel> <!-- TODO - hide logic to add-->
+      <title>Other Alerts</title>
+      <table id="tbl_other_alerts">
+        <search base="basesearch">
+          <query> eval products_to_search="Authentication|VPN|Radius Authentication|Sophos Endpoint Protection|Windows Defender|CrowdStrike|Defender ATP|AWS|Google Workspace|O365|Email|Network Compromise|Cisco IOS|Cisco Meraki|Vulnerability|Windows|AD|Credential Compromise|Ransomware|Linux|F5 BIGIP|Oracle|MSSQL"
```

Comment:
same way product names to change as we have in cs_product_list dict

Reply:
Done